common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -89,6 +89,7 @@
      * Remove all keys that are the command reference from the executing command map.  Since if they key has
      * been changed, remove will not work, construct a new map and add all keys that are not identically
      * that key into the new map.
```

Comment:
[The string](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1060/commits/a0beb2314fbb70070530e88a4884a2190a9c35f3) Adam added in the other pr is gone.

Reply:
I'll `cherry-pick` that in, thanks @rpdome

Reply:
e8f3223dd14c48c78fa198d570948c0aebbd20c9